internal/runtime/runtime.go

```diff
@@ -75,6 +73,7 @@ type GolangRuntime struct {
 	pipelines     map[string]*interfaces.FunctionPipeline
 	isBusyCopying sync.Mutex
 	storeForward  storeForwardInfo
+	lc            logger.LoggingClient
```

Comment:
does this change break anything?

Reply:
No, this code is all internal and it is additive.